test/models/config_test.rb

```diff
@@ -94,5 +94,25 @@ class ConfigTest < ActiveSupport::TestCase
         assert(Config.start_day.equal? :tuesday)
       end
     end
+
+    describe '#hide_practical_support?' do
+      it 'can return true' do
+        c = Config.find_or_create_by(config_key: 'hide_practical_support')
```

Comment:
Any reason not to do all this on create like we do in the system test, esp since we're not really changing the object state? Would work either way, but would save us assigning the local variable and a database call (and I like the consistency). I'm good either way, let me know yr thoughts.